script/tool/lib/src/version_check_command.dart

```diff
@@ -141,8 +150,26 @@ class VersionCheckCommand extends PackageLoopingCommand {
   static const String _breakingChangeJustificationMarker =
       '## Breaking change justification';
 
+  /// The string that must be at the start of a line in [_changeDescriptionFile]
+  /// to allow skipping a version change for a PR that would normally require
+  /// one.
+  static const String _missingVersionChangeJustificationMarker =
+      'No version change:';
+
+  /// The string that must be at the start of a line in [_changeDescriptionFile]
+  /// to allow skipping a CHANGELOG change for a PR that would normally require
+  /// one.
+  static const String _missingChangelogChangeJustificationMarker =
+      'No CHANGELOG change:';
+
   final PubVersionFinder _pubVersionFinder;
 
+  late GitVersionFinder _gitVersionFinder;
+  late String _mergeBase;
+  late List<String> _changedFiles;
```

Comment:
These can be late final since they are only set once, no?

Reply:
Good call, done.